templates/vite/src/routeTree.gen.ts

```diff
@@ -0,0 +1,118 @@
+/* prettier-ignore-start */
+
+/* eslint-disable */
+
+// @ts-nocheck
+
+// noinspection JSUnusedGlobalSymbols
+
+// This file is auto-generated by TanStack Router
+
+import { createFileRoute } from '@tanstack/react-router'
+
+// Import Routes
+
+import { Route as rootRoute } from './routes/__root'
+
+// Create Virtual Routes
+
+const ScriptLazyImport = createFileRoute('/script')()
+const PredicateLazyImport = createFileRoute('/predicate')()
+const FaucetLazyImport = createFileRoute('/faucet')()
+const IndexLazyImport = createFileRoute('/')()
+
+// Create/Update Routes
+
+const ScriptLazyRoute = ScriptLazyImport.update({
+  path: '/script',
+  getParentRoute: () => rootRoute,
+} as any).lazy(() => import('./routes/script.lazy').then((d) => d.Route))
+
+const PredicateLazyRoute = PredicateLazyImport.update({
+  path: '/predicate',
+  getParentRoute: () => rootRoute,
+} as any).lazy(() => import('./routes/predicate.lazy').then((d) => d.Route))
+
+const FaucetLazyRoute = FaucetLazyImport.update({
+  path: '/faucet',
+  getParentRoute: () => rootRoute,
+} as any).lazy(() => import('./routes/faucet.lazy').then((d) => d.Route))
+
+const IndexLazyRoute = IndexLazyImport.update({
+  path: '/',
+  getParentRoute: () => rootRoute,
+} as any).lazy(() => import('./routes/index.lazy').then((d) => d.Route))
+
+// Populate the FileRoutesByPath interface
+
+declare module '@tanstack/react-router' {
+  interface FileRoutesByPath {
+    '/': {
+      id: '/'
+      path: '/'
+      fullPath: '/'
+      preLoaderRoute: typeof IndexLazyImport
+      parentRoute: typeof rootRoute
+    }
+    '/faucet': {
+      id: '/faucet'
+      path: '/faucet'
+      fullPath: '/faucet'
+      preLoaderRoute: typeof FaucetLazyImport
+      parentRoute: typeof rootRoute
+    }
+    '/predicate': {
+      id: '/predicate'
+      path: '/predicate'
+      fullPath: '/predicate'
+      preLoaderRoute: typeof PredicateLazyImport
+      parentRoute: typeof rootRoute
+    }
+    '/script': {
+      id: '/script'
+      path: '/script'
+      fullPath: '/script'
+      preLoaderRoute: typeof ScriptLazyImport
+      parentRoute: typeof rootRoute
+    }
+  }
+}
+
+// Create and export the route tree
+
+export const routeTree = rootRoute.addChildren({
+  IndexLazyRoute,
+  FaucetLazyRoute,
+  PredicateLazyRoute,
+  ScriptLazyRoute,
+})
+
+/* prettier-ignore-end */
+
+/* ROUTE_MANIFEST_START
```

Comment:
Nit, can we make this a `manifest.json` file for json validation.

Reply:
https://github.com/FuelLabs/fuels-ts/blob/5be643bd4ca8b4472386d2c47ac3a896f39b75c7/templates/vite/src/routeTree.gen.ts#L9